src/scene/vcPOI.cpp

```diff
@@ -873,19 +876,12 @@ void vcPOI::HandleImGui(vcState *pProgramState, size_t *pItemID)
     vdkProjectNode_SetMetadataString(m_pNode, "textSize", pTemp);
   }
 
-  // Handle hyperlinks
-  const char *pHyperlink = m_metadata.Get("hyperlink").AsString();
-  if (pHyperlink != nullptr)
-  {
-    ImGui::TextWrapped("%s: %s", vcString::Get("scenePOILabelHyperlink"), pHyperlink);
-    if (udStrEndsWithi(pHyperlink, ".png") || udStrEndsWithi(pHyperlink, ".jpg"))
-    {
-      ImGui::SameLine();
-      if (ImGui::Button(vcString::Get("scenePOILabelOpenHyperlink")))
-        pProgramState->pLoadImage = udStrdup(pHyperlink);
```

Comment:
@OsiJr should this continue to happen? I was under the impression that we were specifically requested to load any images in Client?

Reply:
Images yes; Turns out that PDFs, movies and other miscellany were being linked as well

Reply:
Actually; yes- you're correct. If we _can_ handle it we should. Ideally these things with Images should be `MediaNodes` though

Reply:
I see, I believe these come from the UDP parser which predates `MediaNodes`, and it wasn't updated when they were added. Are they definitely the same feature? In Geoverse, they're a label that opens a picture - I think it might have opened through the scene viewer too by clicking on the label.

Reply:
am I good to remove this code or do you guys want it in?

Reply:
ok ive put it back in. just fyi the button never activates since the metadata never has "hyperlink" in it

Reply:
@Jossos There seems to be a misunderstanding here - we weren't asking for the code to be put back, we were asking for the functionality to return. The functionality being that if the URI is for an image, load the image in Client using the code commented on here, otherwise call out to the OS as you've done in this PR.